onnxruntime/core/providers/webnn/builders/impl/max_min_op_builder.cc

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Intel Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/common.h"
+#include "core/providers/shared/utils/utils.h"
+#include "core/providers/webnn/builders/helper.h"
+#include "core/providers/webnn/builders/model_builder.h"
+#include "core/providers/webnn/builders/op_builder_factory.h"
+
+#include "base_op_builder.h"
+
+namespace onnxruntime {
+namespace webnn {
+
+class MaxMinOpBuilder : public BaseOpBuilder {
+  // Add operator related.
+ private:
+  Status AddToModelBuilderImpl(ModelBuilder& model_builder, const Node& node,
+                               const logging::Logger& logger) const override ORT_MUST_USE_RESULT;
+
+  // Operator support related.
+  bool IsOpSupportedImpl(const InitializedTensorSet& initializers, const Node& node,
+                         WebnnDeviceType /* device_type */, const logging::Logger& logger) const override;
+};
+
+// Add operator related.
+
+Status MaxMinOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder,
+                                                 const Node& node,
+                                                 const logging::Logger& logger) const {
```

Comment:
```suggestion
Status MaxMinOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder,
                                              const Node& node,
                                              const logging::Logger& logger) const {
```

[nit] Indent misaligned.


Note some other parts of ORT don't encounter this annoying misalignment tedium, like WinML which would align them so:

```
Status MaxMinOpBuilder::AddToModelBuilderImpl(
  ModelBuilder& model_builder,
  const Node& node,
  const logging::Logger& logger
) const {
```

Or the DML EP:

```
Status MaxMinOpBuilder::AddToModelBuilderImpl(
    ModelBuilder& model_builder,
    const Node& node,
    const logging::Logger& logger
) const
{
```

But the WebNN EP inherits the default clang-format.

Reply:
Below too for `MaxMinOpBuilder::IsOpSupportedImpl(`.